### PR TITLE
Synchronizer broker registration & tender plan handler fix

### PR DESCRIPTION
**Pull request by tatianaIljina**
1. TenderPlanHandler CreateFeomCurrent select only active positions
2. Tenders.Sunchronization.TenderPlan.Synchronizer uses RegistrationHelper in MessageBroker.Extensions

### CODE REVIEW

File: Synchronization/Tenders.Synchronization.TenderPlan.Synchronizer/Services/TenderPlanHandler.cs [hunk 1]
```diff
@@ -143,12 +143,12 @@ private ITenderPlan MapFromDto(ITenderPlanDto model)
             return result;
         }
 
-        private void MapPositionsFromDto(ITenderPlan tenderPlan, IEnumerable<Abstraction.ITenderPlanPositionDto> positions)
+        private void MapPositionsFromDto(ITenderPlan tenderPlan, IEnumerable<MessageBroker.Extensions.TenderPlan223DtoTopic.ITenderPlanPositionDto> positions)
```
Comment (DamienVStone) [inline Synchronization/Tenders.Synchronization.TenderPlan.Synchronizer/Services/TenderPlanHandler.cs:146]: why namespace not in **using** section?

Comment (tatianaIljina) [reply Synchronization/Tenders.Synchronization.TenderPlan.Synchronizer/Services/TenderPlanHandler.cs:146]: Due to the ambiguity between the two names (Tenders.Core.Abstraction.Model)
Should I use smth like this in using section?
**using ITenderPlanPositionDto = Tenders.Synchronization.TenderPlan.Synchronizer.Abstraction.ITenderPlanPositionDto;**

Comment (DamienVStone) [reply Synchronization/Tenders.Synchronization.TenderPlan.Synchronizer/Services/TenderPlanHandler.cs:146]: Rename Tenders.Core.Abstraction.Model.ITenderPlanPositionDto to Tenders.Core.Abstraction.Model.ITenderPlanPositionViewModel

Comment (DamienVStone) [reply Synchronization/Tenders.Synchronization.TenderPlan.Synchronizer/Services/TenderPlanHandler.cs]: I specifically indicated the namespace

**Pull request merged**